.github/workflows/review.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: AI Code Reviewer
-        uses: lfsevergnini/ai-codereviewer@main
+        uses: PierreGode/GPTcode-reviwer@main
```

Comment:
The action 'PierreGode/GPTcode-reviwer@main' may not be stable or widely used compared to 'lfsevergnini/ai-codereviewer@main'. Consider verifying the reliability and support of this action before using it.